docs/kh2/file/type/00battle.md

```diff
@@ -481,7 +481,11 @@ Data on summons.
 
 | Offset | Type  | Description
 |--------|-------|--------------
-| 00     | 2B[7] | ???
+| 0     | 2B | ???
```

Comment:
For the sake of consistency, could you change `2B` as `short`/`ushort` and `50B` as `byte[50]`?

Reply:
Done